### PR TITLE
docs(IA): singular components

### PR DESCRIPTION
**Pull request by cathysiller**
JIRA: SURF-1634 and SURF-1621

### LGTM's
- [x] Dev LGTM
- [x] Zoom LGTM

Things done in this p/r:
- Updated components from plural to singular (ie accordions --> accordion)
- Relabeled "Tabs" to "Tabset"
- Reordered left-hand navigation links based on path, not label ie., <hx-radio> before <hx-radio-control> before <hx-radio-set>
- updated breadcrumbs in test pages
- NOTE: we are NOT updating the `{% set contentClasses ... ` plural to singular in the test pages for this p/r

### CODE REVIEW

**Comment by HelixUI** [comment]
Deploy preview for *helix-ui* processing.

Building with commit f7900fad2e0d5eae595937f20c3cf81f0f0df25e

https://app.netlify.com/sites/helix-ui/deploys/5cc9f186367f7f00088ea13e

File: docs/components/radio/test.html [hunk 1]
```diff
@@ -74,16 +77,19 @@
   </hx-radio-set>
 {% endmacro %}
 
+<<<<<<< HEAD:docs/components/radio/test.html
```
Comment (CITguy) [inline docs/components/radio/test.html]: Please correct this merge conflict.

File: docs/components/file/test-file-selector.html [hunk 2]
```diff
@@ -1,16 +1,12 @@
...
-  <section>
-    <nav class="hxBreadcrumb">
-      <a href="components/files">Files</a>
-      <hx-icon class="delimiter" type="angle-right"></hx-icon>
-      <a href="#">{{page.title}}</a>
-    </nav>
-  </section>
 
+{% block content %}
   <h2>Upload File Selectors</h2>
```
Comment (CITguy) [inline docs/components/file/test-file-selector.html]: Please update this heading, given you updated the heading at line 51.

File: docs/_data/nav.json5 [hunk 3]
```diff
@@ -84,12 +84,12 @@
...
             { label: '<hx-progress>', path: 'hx-progress' },
+            { label: '<hx-radio>', path: 'hx-radio' },
             { label: '<hx-radio-control>', path: 'hx-radio-control' },
             { label: '<hx-radio-set>', path: 'hx-radio-set' },
-            { label: '<hx-radio>', path: 'hx-radio' },
             { label: '<hx-reveal>', path: 'hx-reveal' },
-            { label: '<hx-search-assistance>', path: 'hx-search-assistance' },
             { label: '<hx-search>', path: 'hx-search' },
+            { label: '<hx-search-assistance>', path: 'hx-search-assistance' },
             { label: '<hx-select-control>', path: 'hx-select-control' },
```
Comment (CITguy) [inline docs/_data/nav.json5]: `<hx-select>` before `<hx-select-control>`

File: docs/components/layout/index.html [hunk 4]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Layouts
+title: Layout
```
Comment (100stacks) [inline docs/components/layout/index.html]: Our singular naming convention breaks down a little here, though not a showstopper. 👍

File: docs/components/tooltip/test.html [hunk 5]
```diff
@@ -1,16 +1,12 @@
...
-{% block content %}
-  <section>
-    <nav class="hxBreadcrumb">
-      <a href="components/tooltips">Tooltips</a>
-      <hx-icon class="delimiter" type="angle-right"></hx-icon>
-      <a href="#">{{page.title}}</a>
-    </nav>
-  </section>
 
+{% block content %}
```
Comment (100stacks) [inline docs/components/tooltip/test.html]: We're missing section `id`s for the **side panel**, though not a major issue.

Comment (CITguy) [reply docs/components/tooltip/test.html]: yup, that'll be handled in a future PR, when we normalize the test pages in preparation for automated testing

**Pull request merged**